web-ui/src/pages/ReceivedRequestsPage.jsx

```diff
@@ -214,84 +284,167 @@ const ReceivedRequestsPage = () => {
         <IconButton
           onClick={() => setReceivedRequestsExpanded(!receivedRequestsExpanded)}
           aria-label="show more"
-          className={receivedRequestsExpanded ? classes.expandOpen : classes.expandClose}
-          size="large">
+          className={
+            receivedRequestsExpanded ? classes.expandOpen : classes.expandClose
+          }
+          size="large"
+        >
           <ExpandMoreIcon />
         </IconButton>
       </div>
       <Divider />
       <Collapse in={!receivedRequestsExpanded} timeout="auto" unmountOnExit>
         <div style={{ marginTop: "1em" }} className="no-requests-message">
-          <Typography variant="body1">{receivedRequests.length} received request{receivedRequests.length === 1 ? "" : "s"} currently hidden</Typography>
+          <Typography variant="body1">
+            {receivedRequests.length} received request
+            {receivedRequests.length === 1 ? "" : "s"} currently hidden
+          </Typography>
         </div>
-
       </Collapse>
       <Collapse in={receivedRequestsExpanded} timeout="auto" unmountOnExit>
-        {isLoading &&
-          <div style={{ marginTop: "1em" }} >
-            {Array.from({ length: 1 })
-              .map((_, index) =>
-                <SkeletonLoader key={index} type="received_requests" />
-              )
-            }
+        {isLoading && (
+          <div style={{ marginTop: "1em" }}>
+            {Array.from({ length: 1 }).map((_, index) => (
+              <SkeletonLoader key={index} type="received_requests" />
+            ))}
           </div>
-        }
-        {!isLoading &&
+        )}
+        {!isLoading && (
           <div className="received-requests-container">
-            {receivedRequests.length === 0 &&
-              <div className="no-requests-message"><Typography variant="body1">No received feedback requests</Typography></div>
-            }
-            {receivedRequests.length > 0 && filteredReceivedRequests.length === 0 &&
-              <div className="no-requests-message"><Typography variant="body1">No matching feedback requests</Typography></div>
-            }
+            {receivedRequests.length === 0 && (
+              <div className="no-requests-message">
+                <Typography variant="body1">
+                  No received feedback requests
+                </Typography>
+              </div>
+            )}
+            {receivedRequests.length > 0 &&
+              filteredReceivedRequests.length === 0 && (
+                <div className="no-requests-message">
+                  <Typography variant="body1">
+                    No matching feedback requests
+                  </Typography>
+                </div>
+              )}
```

Comment:
This could be collapsed into a single: `receivedRequests.length === 0 || (receivedRequests.length > 0 && filteredReceivedRequests.length === 0) && ...`